cmd/endpoint.go

```diff
@@ -182,12 +182,26 @@ func (endpoints EndpointList) SetHTTP() {
 func NewEndpointList(args ...string) (endpoints EndpointList, err error) {
 	// isValidDistribution - checks whether given count is a valid distribution for erasure coding.
 	isValidDistribution := func(count int) bool {
-		return (count >= 4 && count <= 16 && count%2 == 0)
+		return (count >= minErasureBlocks && count <= maxErasureBlocks && count%2 == 0)
+	}
+
+	// getValidErasureBlocks returns a string containing the valid values for erasure coding.
+	getValidErasureBlocks := func() string {
+		list := ""
+		for blocks := minErasureBlocks; blocks <= maxErasureBlocks; blocks += 2 {
+			list += fmt.Sprintf("%d", blocks)
+			if blocks < maxErasureBlocks-2 {
+				list += ", "
+			} else if blocks == maxErasureBlocks-2 {
+				list += " or "
+			}
+		}
+		return list
```

Comment:
As this function is called at startup only, its OK to be expensive.    If its also a concern, we could have `validErasureBlocks` along with `{min|max}ErasureBlocks`.

What I understood from this PR is that, `{min|max}ErasureBlocks` can be changed in later time and `NewEndpointList()` would work without a change.   To extend that I would like to move out `divisible by 2` logic from `NewEndpointList()`.